plugins/github-issues/src/api/gitHubIssuesApi.ts

```diff
@@ -151,20 +189,50 @@ export const gitHubIssuesApi = (
   return { fetchIssuesByRepoFromGitHub };
 };
 
+function formatFilterValue(value: IssuesFilters[keyof IssuesFilters]): string {
+  if (Array.isArray(value)) {
+    return `[ ${value.map(formatFilterValue).join(', ')}]`;
+  }
+
+  return typeof value === 'string' ? `\"${value}\"` : `${value}`;
+}
+
+function createFilterByClause(filterBy?: IssuesFilters): string {
+  if (typeof filterBy === 'undefined') {
```

Comment:
If you want to check for it being unset or empty you can do.
```suggestion
  if (!filterBy) {
```